### PR TITLE
enh(release): add Centreon Web 21.04.9 RN

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target serie

- [ ] 20.04.x
- [ ] 20.10.x
- [x] 21.04.x
- [ ] 21.10.x
- [ ] 22.04.x (master)

### CODE REVIEW

File: en/releases/centreon-core.md [hunk 1]
```diff
@@ -19,6 +19,18 @@ If you have feature requests or want to report a bug, please go to our
...
+#### Bug Fixes
+
+- [Authentication] Fixed PHP error when debug is enabled with OIDC authentication
+- [CLAPI] Fixed LDAP configuration ID for export and import
+- [Configuration] Fixed Centreon Broker configuration that could be empty or deleted
+- [Configuration] Fixed the list of host template that was not available if the database name was different from the default
+- [UX] Fixed display of tooltips in configuration forms
+- [UX] Non admin user do not have the same submenu subsections
+- [UX] Removed "Animation effects" option
+
```
Comment (mdebauchez) [inline en/releases/centreon-core.md:33]: Maybe MON-11139 : [Dependency to upgrade on : centreon/centreon] - composer prod. bump nelmio/cors-bundle from 1.5.6 to 2.1.1 should appear

File: en/releases/centreon-core.md [hunk 2]
```diff
@@ -19,6 +19,18 @@ If you have feature requests or want to report a bug, please go to our
...
+#### Bug Fixes
+
+- [Authentication] Fixed PHP error when debug is enabled with OIDC authentication
+- [CLAPI] Fixed LDAP configuration ID for export and import
+- [Configuration] Fixed Centreon Broker configuration that could be empty or deleted
+- [Configuration] Fixed the list of host template that was not available if the database name was different from the default
+- [UX] Fixed display of tooltips in configuration forms
+- [UX] Non admin user do not have the same submenu subsections
+- [UX] Removed "Animation effects" option
```
Comment (mdebauchez) [inline en/releases/centreon-core.md:32]: This ticket is not in the release

Comment (mdebauchez) [reply en/releases/centreon-core.md:32]: This one is missing : MON-11477 [VALIDATION@FEEDBACK] Php warnings on dependencies

**Review by mdebauchez — CHANGES_REQUESTED** [review]

File: en/releases/centreon-core.md (hunk 2 above)
Comment (lpinsivy) [reply en/releases/centreon-core.md:32]: no this is technical

File: en/releases/centreon-core.md (hunk 1 above)
Comment (lpinsivy) [reply en/releases/centreon-core.md:33]: no this is technical

File: en/releases/centreon-core.md (hunk 2 above)
Comment (mdebauchez) [reply en/releases/centreon-core.md:32]: No I wanted to say that the ticket _[UX] Removed "Animation effects" option_ does not concern the 21.04.9

**Review by mdebauchez — CHANGES_REQUESTED** [review]

**Review by mdebauchez — APPROVED** [review]

File: en/releases/centreon-core.md [hunk 3]
```diff
@@ -19,6 +19,18 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 21.04.9
+
```
Comment (chgautier) [inline en/releases/centreon-core.md:23]:
```suggestion

Release date: `November 29, 2021`

```

File: fr/releases/centreon-core.md [hunk 4]
```diff
@@ -19,6 +19,18 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 21.04.9
+
```
Comment (chgautier) [inline fr/releases/centreon-core.md:23]:
```suggestion

Release date: `November 29, 2021`

```

**Review by chgautier — APPROVED** [review]

**Pull request merged**